src/libs/dutil/test/DUtilUnitTest/DUtilTests.cpp

```diff
@@ -21,7 +21,12 @@ namespace DutilTests
             Dutil_TraceSetLevel(REPORT_DEBUG, FALSE);
 
             Action^ action = gcnew Action(this, &DUtil::CallDutilTraceErrorSource);
-            Assert::Throws<Exception^>(action);
+            // xxxxx The following line is ambiguous when used with the current version of Xunit.
+            // xxxxx Xunit::Assert::Throws<Exception^>(action);
+            // xxxxx It has therefore been, at least temporarily, replaced by the next line of code.
+            // xxxxx See the comments in WixBuildTools.WixAssert for details.
```

Comment:
We don't write comments like `// xxxxx {Text}` in WiX. Please remove the `xxxxx`s. Also, we would normally add a blank line above the comments and no blank line between the comment and its code but you have reversed that here.